polaris-react/src/components/Banner/Banner.tsx

```diff
@@ -79,14 +79,12 @@ export const Banner = forwardRef<BannerHandles, BannerProps>(function Banner(
   );
 
   let headingMarkup: React.ReactNode = null;
-  let headingID: string | undefined;
 
   if (title) {
-    headingID = `${id}Heading`;
     headingMarkup = (
-      <div className={styles.Heading} id={headingID}>
```

Comment:
Removing id is likely a breaking change. I imagine this was added for analytics or tracking of some kind. Makes me a bit uncomfortable to remove it but I want to

Reply:
There is no way for teams to know the ID as it's generated with `useUniqueId`. I think this should be safe to remove. Other ID's though this could be the case.

Reply:
@BPScott do you remember why banners needed unique ids?